internal/view/pod.go

```diff
@@ -337,6 +356,25 @@ func attachIn(a *App, path, co string) {
 	}
 }
 
+func computeDebugArgs(path, co string, kcfg *string, os string, image string) []string {
+	args := make([]string, 0, 15)
+	args = append(args, "debug", "-it")
+
+	ns, po := client.Namespaced(path)
+	if ns != client.AllNamespaces {
+		args = append(args, "-n", ns)
+	}
+	args = append(args, po)
+	if kcfg != nil && *kcfg != "" {
+		args = append(args, "--kubeconfig", *kcfg)
+	}
+
+	args = append(args, "--target", co)
+	args = append(args, fmt.Sprintf("--image=%s", image))
```

Comment:
If  we have multiple containers in 1 pod, do we need `-c`?

Reply:
```
    -c, --container='':
	Container name to use for debug container.
```

I don't think we need to name the debug container here. I'm pretty sure it just gets a randomized name from kubectl debug.

Target is the option that ensures we connect to the selected container (whether or not there are multiple in the pod).

Reply:
I'm not totally sure, but I think there are (at least) 2 different ways how to debug existing containers. 
Only one concern about `target` from https://kubernetes.io/docs/tasks/debug/debug-application/debug-running-pod/#ephemeral-container
`
Note: The --target parameter must be supported by the [Container Runtime](https://kubernetes.io/docs/setup/production-environment/container-runtimes). When not supported, the Ephemeral Container may not be started, or it may be started with an isolated process namespace so that ps does not reveal processes in other containers.
`

Reply:
Not sure if @pinarruiz wants to weigh in, but the non-ephemeral option (`--copy-to`) isn't very interesting to me since you get a new container rather than a way to inspect the running container.

Reply:
As best I can tell ephemeral containers should be pretty well supported at this point. Let me know if there's anything else you'd like to see changed here before merging. Thanks!

Reply:
I would say the copy to option is interesting, as not everyone has support for ephemeral containers enabled, might be nice to include in the little menu i was refering at https://github.com/derailed/k9s/pull/1850#issuecomment-1309900118, what do you think?

Reply:
@matschaffer-roblox @pinarruiz  @slimus Thank you all for weighting in!!  I think we need to support both options ie ephemeral co and copy-to especially when distroless containers are at play. Also my understanding with ephemeral co, is that once launched you can't nuke them (easily...). I think one important aspect of this feature is how we leave out the state of the cluster once the debug session is over as these could yield major security holes on the cluster. Also we need to make it easy for folks to start debug sessions from k9s without being exposed to too many options... Hence I think we should provide a dialog upon invocation to allow user to specify:
1. Ephemeral co or pod
2. The image to use
3. The command to use
Minimally, if we launch a debug session we need to make sure we nuke it on the way out.